### ISSUE TITLE
Feature request: show the number of dependencies

### ISSUE THREAD

**Issue by JosephusPaye** [styfle/packagephobia#720]
Thanks for this, it's a great tool that I use frequently.

When comparing two packages, I find that the first thing I want to know after the publish and install sizes is the number of dependencies. For this I've had to use the npm website together with Package Phobia. It would be nice to have the number of dependencies displayed in Package Phobia.

It could look something like the following:

![Screen shot of Package Phobia showing the package ava with number of dependencies](https://user-images.githubusercontent.com/5924865/113469981-b5743400-949d-11eb-97f0-9ee209ed8fe4.png)

**Comment by styfle** [comment]
Good idea! Indeed, the reason for linking Package Phobia to npm is for those additional details.

In #124 we added the file count. So we could also add the dependency count on hover.

I kinda want to avoid too many numbers on the main page here because it can become noisy.
